src/components/Form/Form.tsx

```diff
@@ -0,0 +1,65 @@
+
+import React from 'react'
+import CheckRadioConcept from './CheckRadioConcept'
+import RangeConcept from './RangeConcept'
+import { CHECK_AND_RADIO,RANGE } from '../../data'
+
+const Modal = ({ message, onClose }: { message: string, onClose: () => void }) => (
+  <div style={{ position: 'fixed', top: '50%', left: '50%', transform: 'translate(-50%, -50%)', backgroundColor: 'white', padding: '20px', zIndex: 1000 }}>
+    <p>{message}</p>
+    <button onClick={onClose}>Close</button>
+  </div>
+);
```

Comment:
since modal could be reused and it isnt really intuitive to have it in `Form.tsx` file please move it to another file